src/features/auth/auth.module.ts

```diff
@@ -0,0 +1,14 @@
+import { Module } from '@nestjs/common';
+import { AuthService } from './auth.service';
+import { PassportModule } from '@nestjs/passport';
+import { LocalStrategy } from './local.strategy';
+import { UserModule } from '../user/user.module';
+import { UserController } from '../user/user.controller';
+import { AuthController } from './auth.controller';
+
+@Module({
+  controllers: [AuthController, UserController],
```

Comment:
Why AuthModule respond for UserController

Reply:
correct